### DISCUSSION TITLE
Using it as a simple String Calculator

### DISCUSSION THREAD

**Discussion by pranav230** [petitparser/dart-petitparser#77] [Q&A] [answered]
I am working on a project where i want to calculate the string as a **Mathematical Expression** like 
`'76+98-47*(2*5)+8'`
How do i use this package to calculate this string output?

**Comment by renggli** [comment, marked as answer]
Did you check out the tutorial that builds a calculator?
https://github.com/petitparser/dart-petitparser#writing-a-more-complicated-grammar

Did you have a look at the examples that includes a calculator?
https://github.com/petitparser/dart-petitparser/tree/master/petitparser/example

**Comment by pranav230** [comment]
Thanks for it
